docs/developers/define-secrets.rst

```diff
@@ -128,25 +124,11 @@ Finally, here is an example of a static secret that needs an additional layer of
      onepassword:
        encoded: true
 
-.. _secret-definition-legacy:
-
-Create the secret generation rules (legacy)
-===========================================
-
-Currently, the Vault secrets for an environment are created using the `installer/generate_secrets.py script <https://github.com/lsst-sqre/phalanx/blob/main/installer/generate_secrets.py>`__.
-That Python script can handle both static secrets that the installer needs to retrieve from 1Password or prompt for, and dynamic secrets that can be generated automatically via the ``SecretGenerator`` class.
-Every application secret must have corresponding code in :file:`generate_secrets.py` to either generate it or prompt for it (which retrieves it from 1Password for SQuaRE-managed environments).
-
-This script is used in conjunction with the `installer/update_secrets.sh script <https://github.com/lsst-sqre/phalanx/blob/main/installer/update_secrets.sh>`__ to populate the Vault secrets for a Phalanx environment.
-
-Since this mechanism is currently being replaced with a new set of tooling, it is not documented here in detail.
-Adding new secrets to this mechanism will require reading the source and following the pattern used by other applications.
-
 Define VaultSecret resources
 ============================
 
-Once a secret is in Vault, you need to create or update a ``VaultSecret`` resource in your application's deployment (typically in its Helm chart).
-See :doc:`write-a-helm-chart` for more details on creating a Helm chart for an application.
+The Phalanx secrets tooling will ensure that the secret is in Vault, but you must still create or update a ``VaultSecret`` resource in your application's deployment (typically in its Helm chart) to tell `Vault Secrets Operator`_ how to create a ``Secret`` that your application can use.
+(See :doc:`write-a-helm-chart` for more details on creating a Helm chart for an application.)
```

Comment:
Are these sorts of parenthetical sentences grammatically correct? I don't think I've seen them in technical writing I've read.

Reply:
I think it's grammatically correct, in the sense that you can put whole sentences in parentheses and that's fine in English grammar, but you are entirely correct that I adore my parenthetical asides probably a little too much. :) In this case, reading this again, I don't think the parentheses add anything, and I'll drop them.